### ISSUE TITLE
[Winterblight] Spectres (ones which have manaburn attacks) actually has all mana burn stuff dealing PURE damage and it's not stated in tooltip

### ISSUE THREAD

**Issue by redledcat** [ChalkyBrush/roshpit-bug-tracker#779]
I was wondering why they recting my warlord in extremely rude and brutal way, and it's just 800k fucking pure damage going in my face in 380 AOE blast.
Just remind warlord has 0 native resists and/or shields of full absorbtion, unlike all other characters, so that's question to balance, BUT player has to be informed why that specte unit inflicts a lot more damage than it should.

**Comment by redledcat** [comment]
You can say, "but you suck in this game" and "bm actually has 12 seconds BKB", but pure AOE spells is the thing which is related only to sea bosses and water spirit right now.
Pure AOE is powerful tool itself, and probably shouldn't be used to counter melee and slow heroes they are already pretty punished come on

**Comment by DeckardCainRPC** [comment]
holy fuck, even the Spectre mobs that explode and cast Invoker's WWW on death? is that piercing BKB and doing retarded pure damage??

**Comment by redledcat** [comment]
No, it does not pierce bkb, otherwise Seinaru or Paladin users would have complained immediately.
Thing is BM is defenceless against pure damage and can only facetank it.
When I've read tooltip I saw nothing special and just thought "okay they burnt my shields of ice Q, going to manage them more carefully". Then they killed me 5 more times and I checked actual code of that blast, and it deals 800k of pure damage without any stones active.
"mana_burn"			"8 1000 8000"
local mana_burn = event.mana_burn*10
local burnDamage = math.min(enemy:GetMana(), mana_burn)*10
